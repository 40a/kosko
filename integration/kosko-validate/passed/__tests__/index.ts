import execa from "execa";
import { dirname } from "path";
import { runCLI } from "../../../run";

const testDir = dirname(__dirname);

let result: execa.ExecaReturns;

beforeEach(async () => {
  result = await runCLI(["validate"], {
    cwd: testDir
  });
});

test("should return status code 0", () => {
  expect(result.code).toEqual(0);
});

test("should print the message", () => {
  expect(result.stderr).toEqual(
    expect.stringContaining("Components are valid")
  );
});
