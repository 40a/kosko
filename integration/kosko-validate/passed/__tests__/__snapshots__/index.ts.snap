// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should print the message 1`] = `"success                      Components are valid"`;
